optimize/util/optimize-commons/src/main/java/io/camunda/optimize/service/util/configuration/security/ResponseHeadersConfiguration.java

```diff
@@ -123,12 +123,18 @@ public Map<String, String> getHeadersWithValues() {
     final Map<String, String> headers = new HashMap<>();
 
     final List<String> strictTransportSecurityHeaderValues = new ArrayList<>();
-    if (getHttpStrictTransportSecurityMaxAge() > 0) {
+    if (getHttpStrictTransportSecurityMaxAge() != null) {
       strictTransportSecurityHeaderValues.add("max-age=" + getHttpStrictTransportSecurityMaxAge());
     }
-    if (getHttpStrictTransportSecurityIncludeSubdomains()) {
+
+    Boolean includeSubDomains = getHttpStrictTransportSecurityIncludeSubdomains();
+    if (includeSubDomains == null) {
+      includeSubDomains = false;
+    }
```

Comment:
Instead adding null checks, could you modify the `getHttpStrictTransportSecurityIncludeSubdomains()` method to return `boolean` instead of `Boolean`?

The only other place this method is used is in a `toString()` method